spanner/spannertest/inmem.go

```diff
@@ -533,11 +548,15 @@ func (s *server) ExecuteSql(ctx context.Context, req *spannerpb.ExecuteSqlReques
 	if err != nil {
 		return nil, err
 	}
-	return &spannerpb.ResultSet{
+	rs := &spannerpb.ResultSet{
```

Comment:
nit: below this line an unkeyed struct is being used which can lead to bugs. Please consisder added a stuct field key for the value.

Reply:
Done